api/openapi/api.yaml

```diff
@@ -676,5 +676,41 @@ paths:
                     ]
         "500":
           description: Check schematic log. 
+      tags:
+        - Schema Operation
+
+  /explorer/get_property_label_from_display_name:
+    get:
+      summary: Converts a given display name string into a proper property label string
+      description: Converts a given display name string into a proper property label string
+      operationId: api.routes.get_property_label_from_display_name
+      parameters:
+        - in: query
+          name: schema_url
+          schema:
+            type: string
+          description: Data Model URL
+          example: >-
+            https://raw.githubusercontent.com/Sage-Bionetworks/schematic/develop/tests/data/example.model.jsonld
+          required: true
+        - in: query
+          name: display_name
+          schema:
+            type: string
+            nullable: false
+          description: The display name to be converted
+          example: MolecularEntity
+          required: true
+        - in: query
+          name: strict_camel_case
+          schema:
+            type: bool
```

Comment:
This `type` here has to be set to `boolean`.  Please Make sure that you run `python3 run_api.py` after adding an endpoint. If you can't see swagger UI, that means there's probably a typo in `api.yaml`